src/Model/Tag.php

```diff
@@ -488,54 +488,86 @@ public static function getURIIdListByTag(string $search, int $uid = 0, int $star
 	 * Returns a list of the most frequent global hashtags over the given period
 	 *
 	 * @param int $period Period in hours to consider posts
+	 * @param int $limit  Number of returned tags
 	 * @return array
 	 * @throws \Exception
 	 */
 	public static function getGlobalTrendingHashtags(int $period, $limit = 10)
 	{
 		$tags = DI::cache()->get('global_trending_tags');
```

Comment:
The cache key name should take the period and limit parameter into account, otherwise the results shown would always be the same regardless of the variable parameters.